source/whats-new.txt

```diff
@@ -59,6 +60,76 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-6.9:
+
+What's New in 6.9
+-----------------
+
+- Support for {+mdb-server+} 3.6 is deprecated and will be removed in a future driver release.
+- The driver natively supports explicit resource management for cursors and the
+  ``MongoClient``, ``ClientSession``, and ``ChangeStream`` objects. On compatible Node.js
+  versions, you can use explicit resource management with the ``cursor.stream()``
+  method and ``GridFSDownloadStream`` class.
+
+  To learn how to use explicit resource management, see the
+  :github:`v6.9.0 Release Notes <mongodb/node-mongodb-native/releases/tag/v6.9.0>` on
+  GitHub. For more information about explicit resource management, see the
+  :github:`ECMAScript Explicit Resource Management Proposal </tc39/proposal-explicit-resource-management>`
+  on GitHub.
+- If you're using Node.js 18.13 or later, you can pass the ``autoSelectFamily`` and
+  ``autoSelectFamilyTimeout`` options to the ``MongoClient`` constructor. When the
+  ``autoSelectFamily`` option is ``true``, the driver automatically selects between
+  an IPv4 or IPv6 connection. The ``autoSelectFamilyTimeout`` option specifies the
+  timeout, in milliseconds, for the driver to select the connection family.
+
+  The following code example shows how to use these options:
+
+  .. code-block:: javascript
+     :copyable: true
+
+     const client = new MongoClient(process.env.MONGODB_URI, { autoSelectFamily: true, autoSelectFamilyTimeout: 100 });
```

Comment:
S: not sure we use an env variable in most other places. In this instance, I think you can omit the connection string parameter altogether and the driver uses localhost 27017. Also, newline where possible to reduce the scroll

Reply:
The API docs says URL is required? I can just use localhost.